Dockerfile

```
@@ -0,0 +1,41 @@
+# Start from the latest golang base image
+FROM golang:1.13 as builder
+
+# Set the Current Working Directory inside the container
+WORKDIR /app
+
+# Copy go mod and sum files
+COPY go.mod go.sum ./
+
+# Download all dependencies. Dependencies will be cached if the go.mod and go.sum files are not changed
+RUN go mod download
+
+# Copy the source from the current directory to the Working Directory inside the container
+COPY . .
+
+# Build the Go app
+RUN CGO_ENABLED=0 go build -o tyk .
```

This should be something like:
```suggestion
RUN CGO_ENABLED=1 go build -tags 'goplugin' -o tyk .
```